js/application.js

```diff
@@ -1,8 +1,6 @@
 var game; // for development to access game object in browser
 
-// Game logic
 $(document).ready(function() {
-  View.prependBuildingButtons(BuildingsList)
-  View.enablePauseButton();
-  game = new Game;
+  View.init();
```

Comment:
Do you think we should inject the `BuildingsList` dependency here, rather than referencing it within the method in the View module?